Gemfile

```diff
@@ -13,8 +13,6 @@ gem "rails", "~> 7.0.0"
 gem "sassc-rails"
 
 group :development do
-  gem "spring"
```

Comment:
It's still in the `Gemfile.lock` since the `rails` gem depends on it, but we no longer load it.